client/reader/full-post/_style.scss

```diff
@@ -217,6 +217,10 @@
 		}
 	}
 
+	table {
+		table-layout: fixed;
```

Comment:
That seems... dangerous: https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout

> Under the "fixed" layout method, the entire table can be rendered once the first table row has been downloaded and analyzed. This can speed up rendering time over the "automatic" layout method, but subsequent cell content may not fit in the column widths provided. Any cell that has content that overflows uses the overflow property to determine whether to clip the overflow content.